bokeh/models/widgets/inputs.py

```diff
@@ -51,6 +51,12 @@ class TextInput(InputWidget):
     """)
 
 
+class PasswordInput(TextInput):
+    ''' Single-line password input widget.
+
+    '''
```

Comment:
Probably worth a mention of what exactly the means, i.e. the value can't be inspected in browser inspectors